triage/do-triage.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# This is a helper script that:
+# 1. tries to infer the name of the previous week's triage report
+# 2. extracts the PARENT commmit from the inferred triage report
+# 3. runs curl against the perf site with the extracted PARENT, piping the output to a file for today
+# 4. does a git checkout of a branch for today's triage
+
+# Infer previous week's triage report: look for files from 6, 7, and 8 days ago.
+# (Usually it will be 7.)
+
+LAST_WEEK_FILE=$(for i in 6 7 8 ; do ls $(printf "%04d-%02d-%02d.md " $(date +%Y) $(date +%m) $(echo $(date +%d) - $i | bc)) 2> /dev/null ; done)
```

Comment:
actually this is totally broken. I was a fool and forgot to account for the first week, during which you need to go back to the previous month.

So I'll just throw this out. Maybe I really *will* re-write in Rust. :)

Reply:
Heh. Apparently on Linux, `date -d "2 days ago" +%Y-%m-%d` will "just work"

Reply:
```suggestion
LAST_WEEK_FILE=$(for i in 6 7 8 ; do ls  $(date -d "$i days ago" +%04Y-%02m-%02d).md ; done)
```